### PR TITLE
feat(components/emphasis): dedicated emphasization component

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**
Emphasization function is duplicated multiple times in forms and components.

**What is the chosen solution to this problem?**
`Emphasis` component has been created to mutualize the emphasization method.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Emphasis/Emphasis.component.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	return parts.map((part, index) => {
+		if (value && part.toUpperCase() === value.toUpperCase()) {
+			return <em key={index} className={theme.highlight}>{part}</em>;
+		}
+		return part;
+	});
+}
+
+function Emphasis(props) {
+	const { value, text } = props;
```
Comment (talend-glorieux) [inline packages/components/src/Emphasis/Emphasis.component.js]: Please don't spread those values. Use them directly `this.props.value` and `this.props.text`

File: packages/components/src/Emphasis/Emphasis.component.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+function emphasise(text, value) {
+	if (!text) {
+		return '';
+	}
+	if (!value) {
+		return text;
+	}
+
+	const parts = text.split(new RegExp(`(${value})`, 'gi')).filter(Boolean);
```
Comment (talend-glorieux) [inline packages/components/src/Emphasis/Emphasis.component.js]: That's a lot of operation in one line. 
I don't really get the purpose of the last filter.

File: packages/components/src/Emphasis/Emphasis.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.highlight {
+	font-weight: 600;
```
Comment (talend-glorieux) [inline packages/components/src/Emphasis/Emphasis.scss]: Please use themes variable.

File: packages/components/src/Emphasis/Emphasis.test.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import { shallow } from 'enzyme';
+import Emphasis from './Emphasis.component';
+
+
+describe('Emphasis', () => {
+	const props = {
+		text: 'The lazy quick brown fox jumps over the lazy dog',
+	};
+
+
```
Comment (talend-glorieux) [inline packages/components/src/Emphasis/Emphasis.test.js]: Extra line.

File: packages/components/src/Emphasis/Emphasis.component.js (hunk 2 above)
Comment (jsomsanith) [inline packages/components/src/Emphasis/Emphasis.component.js]: To eliminate empty values I guess

Comment (jsomsanith) [inline packages/components/src/Emphasis/Emphasis.component.js]: But yes that should be more explicit

File: packages/components/src/Emphasis/Emphasis.scss (hunk 3 above)
Comment (jsomsanith) [inline packages/components/src/Emphasis/Emphasis.scss]: Set a theme var, $font-weight-bold
Set it to `bold` to fit search guideline

**Pull request merged**